src/main/java/net/smartcosmos/dto/metadata/MetadataResponse.java

```diff
@@ -19,26 +24,34 @@
     @Setter(AccessLevel.NONE)
     private int version = VERSION; // just in case there is a default constructor sometime
 
-    private final String dataType;
-    private final String entityReferenceType;
-    private final String key;
-    private final Long lastModifiedTimestamp;
-    private final String rawValue;
-    private final String referenceUrn;
+    @ApiModelProperty("The URN of the metadata entity.")
     private final String urn;
-    private final String moniker;
+
+    @ApiModelProperty("The TYPE of the entity the metadata is related to.")
+    private final String ownerType;
+
+    @ApiModelProperty("The URN of the entity the metadata is related to.")
+    private final String ownerUrn;
+
+    @ApiModelProperty("The collection of metadata to create.")
+    private Map<String, Object> metadata;
```

Comment:
Just a reminder: This needs to be transformed in the REST layer. Otherwise, we would get a response like

```
{
    "ownerType": "...",
    "ownerUrn": "...",
    "metadata: {
        "key1": "value1",
        "key2": "value2"
    }
}
```